### PR TITLE
[Tests-only]  Refactoring webui tests so that they use correct sketeton files

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
refactor steps  with appropriate skeleton files.
Following suites have been refactored:
- webUISharingAcceptShares
- webUISharingAcceptSharesToRoot
- webUISharingFilePermissionMultipleUsers
- webUISharingFilePermissionsGroups
- webUISharingFolderAdvancedPermissionMultipleUsers
- webUISharingFolderAdvancedPermissionsGroups
- webUISharingFolderPermissionMultipleUsers
- webUISharingFolderPermissionsGroups

## Related Issue

- Part of owncloud/QA#659

## How Has This Been Tested?

- :robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests
Refactor steps with appropriate skeleton files
## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13877/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13886/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13898/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13932/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13932/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13936/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13936/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13984/

<p>



























</p></details>

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm 👍🏼

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14037/

<p>






</p></details>

File: tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature [hunk 1]
```diff
@@ -48,9 +50,11 @@ Feature: accept/decline shares coming from internal users
     Given these groups have been created:
       | groupname |
       | grp1      |
+    And user "Alice" has created file "/testimage.jpg"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature:53]: IMO
```suggestion
    And user "Alice" has uploaded file "testavatar.jpg" to "testavatar.jpg"
```

File: tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature [hunk 2]
```diff
@@ -122,7 +133,9 @@ Feature: accept/decline shares coming from internal users
 
   @ocis-product-276
   Scenario: accept a previously declined share
-    Given user "Alice" has shared file "lorem.txt" with user "Brian"
+    Given user "Alice" has created file "lorem.txt"
+    And user "Alice" has created file "testimage.jpg"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature:137]: same as above

File: tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature [hunk 3]
```diff
@@ -136,7 +149,9 @@ Feature: accept/decline shares coming from internal users
 
   @skip @issue-4102
   Scenario: delete an accepted share
-    Given user "Alice" has shared file "lorem.txt" with user "Brian"
+    Given user "Alice" has created file "lorem.txt"
+    And user "Alice" has created file "testimage.jpg"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature:153]: same here

File: tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature [hunk 4]
```diff
@@ -171,18 +188,20 @@ Feature: accept/decline shares coming from internal users
   @skip @yetToImplement
   Scenario: User-based accepting is enabled while global is enabled
     Given the setting "Automatically accept new incoming local user shares" in the section "Sharing" has been enabled
+    And user "Brain" has created folder "/simple-folder"
+    And user "Brain" has created file "/testimage.jpg"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature:192]: same here

File: tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature [hunk 5]
```diff
@@ -155,14 +170,16 @@ Feature: accept/decline shares coming from internal users
   @skip @yetToImplement
   Scenario: User-based accepting is disabled while global is enabled
     Given the setting "Automatically accept new incoming local user shares" in the section "Sharing" has been enabled
+    And user "Brain" has created folder "/simple-folder"
+    And user "Brain" has created file "/testimage.jpg"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature:174]: same here

File: tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature [hunk 6]
```diff
@@ -55,72 +59,79 @@ Feature: accept/decline shares coming from internal users
...
-    Then folder "simple-folder (2)" shared by "Alice Hansen" should be in "Declined" state on the webUI
-    And file "testimage (2).jpg" shared by "Alice Hansen" should be in "Declined" state on the webUI
+    Then folder "simple-folder" shared by "Alice Hansen" should be in "Declined" state on the webUI
+    And file "testimage.jpg" shared by "Alice Hansen" should be in "Declined" state on the webUI
 
 
   Scenario: unshare auto-accepted shares
     Given the setting "shareapi_auto_accept_share" of app "core" has been set to "yes"
+    And user "Alice" has created folder "/simple-folder"
+    And user "Alice" has created file "/testimage.jpg"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature:112]: same here

File: tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature [hunk 7]
```diff
@@ -55,72 +59,79 @@ Feature: accept/decline shares coming from internal users
...
-    Then folder "simple-folder (2)" shared by "Alice Hansen" should be in "Declined" state on the webUI
-    And file "testimage (2).jpg" shared by "Alice Hansen" should be in "Declined" state on the webUI
+    Then folder "simple-folder" shared by "Alice Hansen" should be in "Declined" state on the webUI
+    And file "testimage.jpg" shared by "Alice Hansen" should be in "Declined" state on the webUI
 
   @smokeTest
   Scenario: Auto-accept shares
     Given the setting "shareapi_auto_accept_share" of app "core" has been set to "yes"
+    And user "Alice" has created folder "/simple-folder"
+    And user "Alice" has created file "/testimage.jpg"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature:74]: same here

File: tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature [hunk 8]
```diff
@@ -55,72 +59,79 @@ Feature: accept/decline shares coming from internal users
...
-    Then folder "simple-folder (2)" shared by "Alice Hansen" should be in "Accepted" state on the webUI
-    And file "testimage (2).jpg" shared by "Alice Hansen" should be in "Accepted" state on the webUI
+    Then folder "simple-folder" shared by "Alice Hansen" should be in "Accepted" state on the webUI
+    And file "testimage.jpg" shared by "Alice Hansen" should be in "Accepted" state on the webUI
 
 
   Scenario: decline auto-accepted shares
     Given the setting "shareapi_auto_accept_share" of app "core" has been set to "yes"
+    And user "Alice" has created folder "/simple-folder"
+    And user "Alice" has created file "/testimage.jpg"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature:92]: same here

File: tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature [hunk 9]
```diff
@@ -194,6 +213,8 @@ Feature: accept/decline shares coming from internal users
   @skip @yetToImplement
   Scenario: Admin disables auto-accept setting again after user enabled personal auto-accept setting
     Given the setting "Automatically accept new incoming local user shares" in the section "Sharing" has been enabled
+    And user "Brain" has created folder "/simple-folder"
+    And user "Brain" has created file "/testimage.jpg"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature:217]: same here

**Review by swoichha — COMMENTED** [review]
lgtm :+1

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRename</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15066/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIDelete</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15067/

<p>

































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>MarkdownEditor</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15067/

<p>































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15067/

<p>



































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUICreate</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15067/

<p>



















































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15067/

<p>





























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15067/

<p>







































































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15067/

<p>
























































































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRename</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15067/

<p>









































































































</p></details>

**Review by phil-davis — APPROVED** [review]

**Pull request merged**